alcs-frontend/src/app/features/card-detail-dialog/card-detail-dialog.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { UserDto } from '../../services/user/user.dto';
+import { UserService } from '../../services/user/user.service';
+import { ApplicationDto, ApplicationPartialDto } from '../application/application.dto';
+import { ApplicationService } from '../application/application.service';
+
+@Component({
+  selector: 'app-card-detail-dialog',
+  templateUrl: './card-detail-dialog.component.html',
+  styleUrls: ['./card-detail-dialog.component.scss'],
+})
+export class CardDetailDialogComponent implements OnInit {
+  assignees: UserDto[] = [];
```

Comment:
Renamed to users, since this is a list of people you can assign not people who are asssigned